src/features/pastelDB/constants/common.ts

```diff
@@ -18,3 +18,15 @@ export const pastelTableNames = {
   totalbalance: 'totalbalance',
   listaddresses: 'listaddresses',
 }
+
+export const statisticsFieldNames = [
+  { name: 'Difficulty', routeName: '/difficultyovertime' },
+  { name: 'Hashrate', routeName: '/hashrateovertime' },
+  { name: 'PSL Price', routeName: '/priceovertime' },
+  { name: 'Rawtransaction', routeName: '/rawtransaction' },
+  { name: 'Txoutsetinfo', routeName: '/txoutsetinfo' },
+  { name: 'Walletinfo', routeName: '/walletinfo' },
+  { name: 'RawMempool', routeName: '/rawmempoolovertime' },
+  { name: 'MempoolSize', routeName: '/mempoolsizeovertime' },
+  { name: 'Miners Revenue', routeName: '/minersrevenue' },
+]
```

Comment:
I think this belongs to statistics feature

Reply:
Currently here only valid the router url - '/difficultyovertime'.
other urls does not exist, thus when the chart card click, App is loading again.
Updated other urls to '/dashboard', and is also resolved large memory problem.
I will update url whenever add new chart.